diff --git a/dist/typescript/ast-nodes.js b/dist/typescript/ast-nodes.js
index 0bdfe29af13b82a17f717c3e3886a769e27268ff..e61caeed3e671c827763a3d349383dcd9f4646ac 100644
--- a/dist/typescript/ast-nodes.js
+++ b/dist/typescript/ast-nodes.js
@@ -5,7 +5,7 @@ import { timerify } from '../util/Performance.js';
 import { EMPTY_TAGS } from './visitors/jsdoc.js';
 const defaultParseOptions = {
     sourceType: 'unambiguous',
-    experimentalRawTransfer: rawTransferSupported(),
+    experimentalRawTransfer: process.env.KNIP_EXPERIMENTAL_RAW_TRANSFER === '1' && rawTransferSupported(),
 };
 const parseFile = (filePath, sourceText) => {
     const ext = extname(filePath);
